src/Category/Category.agda

```diff
@@ -0,0 +1,22 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- A definition of category. 
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Category.Category where
+
+open import Level using (suc)
+open import Relation.Binary using (Reflexive; TransFlip; Rel)
+
+record Category ℓ : Set (suc (suc ℓ)) where
```

Comment:
The records `Category` and `IsCategory` don't obey the conventions found elsewhere in the library. For instance if I'm understanding it's purpose correctly then `Category` should probably be called `RawCategory`. `IsCategory` shouldn't be parameterised by `RawCategory`, instead it should be parameterised by it's components. There should then be a new record `Category` that wraps the components along with a `IsCategory` field.